src/tracer.js

```diff
@@ -70,17 +70,19 @@ export default class Tracer {
             // Normalize the argument so the implementation is always provided
             // an associative array of fields.
             if (arguments.length === 1) {
-                if (typeof nameOrFields === 'string') {
-                    fields = {
-                        operationName : nameOrFields,
-                    };
-                } else {
+                if (typeof nameOrFields === 'object') {
                     fields = nameOrFields;
+                    nameOrFields = fields.operationName;
+                } else {
+                    fields = {};
                 }
-            } else {
-                fields.operationName = nameOrFields;
             }
-            spanImp = this._imp.startSpan(fields);
+            spanImp = this._imp.startSpan({
+                operationName: nameOrFields,
+                parent: fields.parent ? fields.parent._imp : undefined,
```

Comment:
It's not very important and I don't feel strongly about this, but I'd rather leave the code as it is if this PR is not addressing a particular issue or defect.

I think there's a *weak* argument it's better to pass the span interface as, within the code implementation can always reference the span's implementation but it cannot do the opposite (i.e. reference the span interface given the span implementation object). Also, the LightStep implementation, for example, [relies on the old behavior](https://github.com/lightstep/lightstep-tracer-javascript/blob/master/src/imp/span_imp.js#L173). The LightStep code would be easy to update if you feel strongly that about this change.